spot_micro_joy/scripts/spotMicroJoystickMove.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+
+import rospy
+from std_msgs.msg import Float32, Bool
+from sensor_msgs.msg import Joy
+from geometry_msgs.msg import Vector3
+from geometry_msgs.msg import Twist
+from math import pi
+
+
+class SpotMicroJoystickControl():
+    BUTTON_GREEN = 0
+    BUTTON_RED = 1
+    BUTTON_YELLOW = 2
+    BUTTON_BLUE = 3
+
+    AXES_ROLL = 0
+    AXES_HEIGHT = 1
+    AXES_YAW = 2
+    AXES_PITCH = 3
+
+    WALK_AXES_FORWARD = 1
+    WALK_AXES_STRAFE = 0
+    WALK_AXES_YAW = 2
+
+    MODE_IDLE = 0
+    MODE_STAND = 1
+    MODE_ANGLE = 2
+    MODE_WALK = 3
+
+    def __init__(self):
+
+        self._angle_cmd_msg = Vector3()
+        self._angle_cmd_msg.x = 0
+        self._angle_cmd_msg.y = 0
+        self._angle_cmd_msg.z = 0
+
+        self._speed_cmd_msg = Vector3()
+        self._speed_cmd_msg.x = 0
+        self._speed_cmd_msg.y = 0
+        self._speed_cmd_msg.z = 0
+
```

Comment:
It looks like speed_cmd is not used at all, so this should be removed. You probably carried this over from my keyboard move command node which also (erroneously) still has this in there.
```suggestion
```

Reply:
I thinking about if i need this structs in this object at all. All variables needed information is delivered in the joystick-callback anyways.

Reply:
Seems like recreating these message is slower than reusing them, so i leave it this way, removed the _speed_cmd_msg